producer/start-daemon-deploy.sh

```diff
@@ -6,5 +6,9 @@ if [[ $LSST_DDS_IP != *"."* ]]; then
   echo "Unset LSST_DDS_IP"
   unset LSST_DDS_IP
 fi
-/home/saluser/repos/ts_sal/bin/make_idl_files.py LOVE
-python -u /usr/src/love/producer/main.py
+
+python -u /usr/src/love/producer/main.py &
+
+pid="$!"
+
+wait ${pid}
```

Comment:
@tribeiro How does this works?

Reply:
Hi @sebastian-aranda , these images are designed to attach to a DDS daemon. When we do that we have to be very careful about how we shut down the running process, otherwise we may corrupt the daemon which the images are attached to. When k8s (and I think docker-compose too) is shutting down a pod (or a container) they will send the term signal to the main process (which, in general, contains pid=0). Because these are subprocesses, they will never receive the kill signal and end up later receiving a term signal afterwards, which corrupts the DDS daemon. To protect agains that I create a signal trap and later propagate it to the running process (whichever as the `pid` shown above).

So, what I do is to create a signal trap, that will capture

Reply:
I understand, great! Thanks for the explanation Tiago